packages/xy-chart/utils/drawLabels.ts

```diff
@@ -14,30 +15,46 @@ export const drawTitle = (
     .attr("y", 30)
     .attr("text-anchor", "middle")
     .text(text);
+  selection
+    .append("defs")
+    .append("clipPath")
+    .attr("id", "clip-circle-title")
+    .append("circle")
+    .attr("r", 11)
+    .attr("cx", "39.5%")
```

Comment:
Relative positions are not suitable for widescreen. Maybe we can calculate the width of `d3.selection` first and use absolute position instead. 

<img width="417" alt="image" src="https://user-images.githubusercontent.com/24653555/173199397-b292c5eb-483c-42d4-ac7d-b579fe8df267.png">

But in server runtime(node.js), we can't call any document element methods. So it needs to take a deep look for a better solution.